components/dashboard/src/service/service.tsx

```diff
@@ -51,7 +53,14 @@ function createGitpodService<C extends GitpodClient, S extends GitpodServer>() {
 function getGitpodService(): GitpodService {
     const w = window as any;
     const _gp = w._gp || (w._gp = {});
-    const service = _gp.gitpodService || (_gp.gitpodService = createGitpodService());
+    const createService = () => {
+        if (window.location.search.includes("mock")) {
```

Comment:
**Thought**: Cool! I realize that this trick could also be used by admins to debug user-specific bugs (e.g. a subscription problem where we don't understand how the Plans page can get into that state -- for these cases, maybe an admin could visit `/?userId=1234` and view the app as user 1234 -- with appropriate audit logging to cross-verify admin activity)